tests/acceptance/settings/SettingOptInCheckboxCest.php

```diff
@@ -288,54 +288,6 @@ public function testOptInCheckboxDisplayLocationOrder(AcceptanceTester $I)
 		$I->seeElementInDOM('.woocommerce-additional-fields #ckwc_opt_in');
 	}
 
-	/**
-	 * Test that the opt in checkbox block is automatically added to the WooCommerce Checkout
-	 * block, cannot be removed and links to the integration settings screen.
-	 *
-	 * @since   1.7.1
-	 *
-	 * @param   AcceptanceTester $I  Tester.
-	 */
-	public function testOptInCheckboxBlockInEditor(AcceptanceTester $I)
```

Comment:
Because tests always run the latest WooCommerce version, the opt-in block isn't now available for testing.  I don't like removing a test, and ideally this test could be run in isolation with a < 8.9.0 WooCommerce version.

If we need to do this, I'll open a new PR to reinstate the test with a separate GitHub workflow action to run this in isolation.

Reply:
@n7studios This might be a good idea since we're branching depending on the WC version, not just dropping support for < 8.9.0 WC versions. How much work to support different versions running? Or in this case, is there a way to disable the specific API method being used so it runs the other code path for this specific test?

Reply:
@noelherrick I've updated the PR to run this test in isolation in WooCommerce 8.4.0. I'd prefer we do that rather than ship a PR that loses test coverage.